gitlab-runner/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BUILD_VERSION
 
-FROM idrinth/api-bench-build:${BUILD_VERSION} AS build
+FROM node:21.7.3-alpine AS build
```

Comment:
this still needs to be updated in about the same way as the framework dockerfile

Reply:
updated file